config/samples/resources/dnsrecordset/dns-routing-policy-primary-backup/compute_v1beta1_computesubnetwork.yaml

```diff
@@ -21,3 +21,17 @@ spec:
   region: us-central1
   networkRef:
     name: dnsrecordset-dep-rp-pb-hc
+
+---
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSubnetwork
+metadata:
+  name: dnsrecordset-dep-rp-pb-hc-proxy
```

Comment:
According to the our naming convention, this should be `dnsrecordset-dep2-rp-pb-hc`. The one above should be `dnsrecordset-dep1-rp-pb-hc`.

Context: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#dependency-naming-convention